core/Db/Schema/Mysql.php

```diff
@@ -393,6 +393,17 @@ public function getTablesCreateSql()
                                       UNIQUE KEY unique_plugin_version_title (`plugin_name`, `version`, `title`(100))                            
                                   ) $tableOptions
             ",
+            'annotations'         => "CREATE TABLE `{$prefixTables}annotations` (
+                                      `id` BIGINT UNSIGNED NOT NULL AUTO_INCREMENT,
+                                      `idsite` INTEGER UNSIGNED NOT NULL,
+                                      `date` DATETIME NOT NULL,
+                                      `note` TEXT NOT NULL,
+                                      `starred` TINYINT(1) NOT NULL DEFAULT 0,
+                                      `user` VARCHAR(100) NOT NULL,
+                                      PRIMARY KEY(`id`),
+                                      INDEX index_id_idsite_date (`id`, `idsite`, `date`)                            
```

Comment:
```suggestion
                                      INDEX index_idsite_date (`idsite`, `date`)
```

With `id` already being the primary key there is no need to add this again to a different index. For queries like `SELECT * WHERE idsite = ?` we need an index without it, otherwise no query will use it.

Reply:
id removed from index